lib/lightning_web/live/workflow_live/components.ex

```diff
@@ -145,16 +146,27 @@ defmodule LightningWeb.WorkflowLive.Components do
         <div class="flex place-content-between items-center space-x-2 pt-1">
           <.errors field={@form[:concurrency]} />
           <div
-            :if={Enum.empty?(@form[:concurrency].errors)}
+            :if={
+              not @project_concurrency_disabled and
+                Enum.empty?(@form[:concurrency].errors)
+            }
             class="text-xs text-slate-500 italic"
           >
-            <%= case @form[:concurrency].value do
-              nil -> "Unlimited (up to max available)"
+            <%= case @form[:concurrency].value || "" do
```

Comment:
I never noticed this in the original implementation, should we try and enforce either `nil` or "positive number" here. I'm wondering why I ever needed to check for a string 🤔.

Reply:
Just simplified a bit the pre-existing code that was chexking both for the string and nil with the same message duplicated. Maybe there was a string due to the form params but as far as I remember the Form module would cover this and set as nil.

Reply:
For future reference the `Form`module emulates the params converting nil values to empty strings.